packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
+let {
+  title,
+  link,
+}: {
+  title: string;
+  link: string;
+} = $props();
```

Comment:
todo: now we recommend to extract the properties to a separate Props interface

interface Props {
 title: string;
 link: string;
}

https://svelte.dev/docs/svelte/$props#Type-safety

Reply:
Fixed.